bsdploy/fabfile_digitalocean.py

```diff
@@ -33,20 +36,21 @@ def bootstrap(**kwargs):
     # overwrite sshd_config, because the DO version only contains defaults
     # and a line explicitly forbidding root to log in
     sudo("""echo 'PermitRootLogin without-password' > /etc/ssh/sshd_config""")
```

Comment:
@igalic i didn't have time yet to test your version of the issue, although i prefer it over the one i came up with myself (#105).

the only real difference i see, is that i upload and overwrite the authorized_keys because i saw that the keys that get installed via cloudinit contain explicit measures to disable login as root. so this PR would not work, if the droplet was created with ssh-keys via the DO GUI.

could you look into that, perhaps?

otherwise i will do some testing later this week and probably propose a solution that uses your approach plus the upload/overwrite of the authorized_keys